packages/x-charts/src/LineChart/LineChart.tsx

```diff
@@ -307,6 +318,10 @@ LineChart.propTypes = {
     slotProps: PropTypes.object,
     slots: PropTypes.object,
   }),
+  /**
+   * If `true`, a loading overlay is displayed.
+   */
```

Comment:
We are missing the default https://deploy-preview-12817--material-ui-x.netlify.app/x/api/charts/line-chart/#line-chart-prop-loading

```suggestion
  /**
   * If `true`, a loading overlay is displayed.
   * @default false
   */
```